src/main/java/build/buildfarm/instance/shard/JedisClusterFactory.java

```diff
@@ -59,11 +59,14 @@ public static Supplier<JedisCluster> create() throws ConfigurationException {
               : configs.getBackplane().getRedisPassword(),
           createJedisPoolConfig());
     }
+    // support "" as redis password.
     return createJedisClusterFactory(
         parseUri(configs.getBackplane().getRedisUri()),
         configs.getBackplane().getTimeout(),
         configs.getBackplane().getMaxAttempts(),
-        configs.getBackplane().getRedisPassword(),
+        configs.getBackplane().getRedisPassword().isEmpty()
```

Comment:
You'll need to do a null check here as well. Just use this. Can you also use the same on line 57 above please.
```
import com.google.common.base.Strings;

Strings.isNullOrEmpty(configs.getBackplane().getRedisPassword()) ? null : configs.getBackplane().getRedisPassword(),
```